forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -2615,9 +2615,9 @@ else if (keyword.startsWith("Protection")) {
             inst.addReplacement(re);
         }
 
-        if (keyword.equals("CARDNAME enters the battlefield tapped.")) {
+        if (keyword.equals("CARDNAME enters tapped.")) {
```

Comment:
This Section can be removed, because "K:CARDNAME enters tapped." doesn't exist anymore?